testdata/steward.toml

```diff
@@ -0,0 +1,14 @@
+[sgx]
+enarx_signer = ["c8dc9fe36caaeef871e6512c481092754c57c2ea999f128282ccb563d1602774"]
```

Comment:
Shouldn't the signer and hash always be the same? @haraldh 

Why are these per-backend settings?

Reply:
signer is the digest of the public key of the signer.

Reply:
hash is the measurement of the keep pages and extra data

Reply:
the keys are for different algorithms for the signature for SGX or SNP, therefore we can't use same key

Reply:
What's the use case for using different signing keys per-backend?

Reply:
the measurement of the keep is of course totally different for the technologies because both the binary contents of the shim differs as well as the methodology of doing the measurement as well as the extra data

Reply:
Hash is always different, since it's the hash of Enarx in memory, not the file on disk. I don't know about the signer though.

Reply:
Ok, got it